pom.xml

```diff
@@ -232,6 +238,12 @@
         <groupId>com.google.guava</groupId>
         <artifactId>guava</artifactId>
         <version>${guava.version}</version>
+        <exclusions>
+          <exclusion>
```

Comment:
This jar's license contains multiple licenses. It seems to be used as test code. So I exclude it.